src/xdocs/config_misc.xml

```diff
@@ -431,17 +431,17 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
         <p>Example #10: Comment within the block tries to describe the next code block.</p>
         <source>
 1   public void foo42() {
-2      int a = 5;
-3      if (a == 5) {
-4         int b;
-5         // it is OK
+2     int a = 5;
+3     if (a == 5) {
+4       int b;
+5       // it is OK
 6      } else if (a ==6) { ... }
 7   }
 8
 9   public void foo43() {
-10     try {
-11        int a;
-12     // Why do we catch exception here? - violation (should have the same indentation as line 11)
+10    try {
+11      int a;
+12        // Why do we catch exception here? - violation (should have the same indentation as line 11)
```

Comment:
> This should be indented between try and int a

This is not done.
Comment should be like:
````
+10    try {
+11      int a;
+12     // Why do we catch exception here? - violation (should have the same indentation as line 11)
````
Like in original code.

Reply:
Done the change